pkg/egressgateway/manager.go

```diff
@@ -31,8 +31,9 @@ import (
 )
 
 var (
-	log      = logging.DefaultLogger.WithField(logfields.LogSubsys, "egressgateway")
-	zeroIPv4 = net.ParseIP("0.0.0.0")
+	log                                 = logging.DefaultLogger.WithField(logfields.LogSubsys, "egressgateway")
+	notFoundGatewaySpecialValueIPv4     = net.ParseIP("0.0.0.0")
+	excludedCIDRGatewaySpecialValueIPv4 = net.ParseIP("0.0.0.1")
```

Comment:
We can already export these as we'll need them in the next PR (to translate these values in the `bpf egress list` command):

```suggestion
        // GatewayNotFoundIPv4 is a special IP value used as gatewayIP in the BPF policy map to indicate no gateway was found for the given policy
	GatewayNotFoundIPv4     = net.ParseIP("0.0.0.0")
	
	// ExcludedCIDRIPv4 is a special IP value used as gatewayIP in the BPF policy map to indicate the entry is for an excluded CIDR and should skip egress gateway
	ExcludedCIDRIPv4 = net.ParseIP("0.0.0.1")
```

Reply:
Since you mentioned bpf egress list, I already added the code to translate it :D. I think it should be ok to do it here even though we are not already dropping the traffic.